src/ape/managers/project.py

```diff
@@ -64,150 +54,47 @@ class ProjectManager:
     :class:`~ape.managers.converters.ConversionManager`.
     """
 
-    compilers: ClassVar[CompilerManager] = injected_before_use()  # type: ignore
-    """
-    The group of compiler plugins for compiling source files. See
-    :class:`~ape.managers.compilers.CompilerManager` for more information.
-    Call method :meth:`~ape.managers.project.ProjectManager.load_contracts` in this class
-    to more easily compile sources.
-    """
-
     networks: ClassVar[NetworkManager] = injected_before_use()  # type: ignore
     """
     The manager of networks, :class:`~ape.managers.networks.NetworkManager`.
     To get the active provide, use
     :py:attr:`ape.managers.networks.NetworkManager.active_provider`.
     """
 
-    def __init__(
-        self,
-        path: Path,
-    ) -> None:
-        self.path = Path(path) if isinstance(path, str) else path
-
     def __repr__(self):
         return "<ProjectManager>"
```

Comment:
Move this to `ProjectAPI` as `f"<{self.__class__.__name__}>"`? Maybe add a field about where it came from or what it's name is or whatever.